### PR TITLE
Trigger development 

### PR DESCRIPTION
**Pull request by gianipez**
- speeded up the MergeTriggerInfo module 
- improved diagnostic messages of a few filter modules
- added the calorimeter hits count trigger path in the OffSpill and ExtractedPosition Menu files
- varied some configuration parameters in the TimeCluster Helix filter modules
- lowered the min threshold of the minBias strawHits and caloHits paths

### CODE REVIEW

**Comment by gianipez** [comment]
tagging @dnbrow01

File: Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_hsFilter.fcl [hunk 1]
```diff
@@ -3,8 +3,8 @@ physics.filters.cprDeMHighPStopTargHSFilter.doHelicityCheck  : true
 physics.filters.cprDeMHighPStopTargHSFilter.helicity      : 1
 physics.filters.cprDeMHighPStopTargHSFilter.minNStrawHits : 15.0
 physics.filters.cprDeMHighPStopTargHSFilter.minHitRatio   : 0.4
-physics.filters.cprDeMHighPStopTargHSFilter.minMomentum   : 80.0
-physics.filters.cprDeMHighPStopTargHSFilter.maxMomentum   : 140.0
+physics.filters.cprDeMHighPStopTargHSFilter.minMomentum   : 70.0
```
Comment (brownd1978) [inline Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_hsFilter.fcl:6]: Should there be a prolog 'minMomentum' that can be used by both cpr and tpr?

File: Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_tcFilter.fcl [hunk 2]
```diff
@@ -1,2 +1,2 @@
 physics.filters.cprDeMHighPStopTargTCFilter.requireCaloCluster : true
-physics.filters.cprDeMHighPStopTargTCFilter.minNStrawHits           : 10.0
+physics.filters.cprDeMHighPStopTargTCFilter.minNStrawHits      : 15.0
```
Comment (brownd1978) [inline Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_tcFilter.fcl:2]: Same comment with minNStrawHits.

File: Trigger/data/tprDeP_highP_stopTarg/main_tprDeP_highP_stopTarg_tcFilter.fcl [hunk 3]
```diff
@@ -1,2 +1,2 @@
 physics.filters.tprDePHighPStopTargTCFilter.requireCaloCluster : false
-physics.filters.tprDePHighPStopTargTCFilter.minNStrawHits : 10.0
```
Comment (brownd1978) [inline Trigger/data/tprDeP_highP_stopTarg/main_tprDeP_highP_stopTarg_tcFilter.fcl:2]: "minNStrawHits" is an int.  Please remove the decimal point.  This applies to all the entries

File: TrkFilters/src/SeedFilter_module.cc [hunk 4]
```diff
@@ -48,7 +48,7 @@ namespace mu2e
         fhicl::Atom<double>             maxD0             {     Name("maxD0"),                   Comment("maxD0             ") };
         fhicl::Atom<double>             minT0             {     Name("minT0"),                   Comment("minT0             ") };
         fhicl::Sequence<std::string>    seedFitFlag       {     Name("seedFitFlag"),             Comment("seedFitFlag       ") , std::vector<std::string>{"SeedOK"}};
-        fhicl::Atom<int>                debugLevel        {     Name("debugLevel "),             Comment("debugLevel        ") , 0};
+        fhicl::Atom<int>                debugLevel        {     Name("debugLevel"),             Comment("debugLevel        ") , 0};
```
Comment (brownd1978) [inline TrkFilters/src/SeedFilter_module.cc:51]: good catch!

**Comment by dnbrow01** [comment]
I'm guessing you accidentally tagged me, intending to tag Dave (LBL) Brown, as this set of commits does not appear to touch any code I've worked on directly.  FWIW, I looked the code over and agree with that Dave's comments.

File: Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_hsFilter.fcl (hunk 1 above)
Comment (gianipez) [reply Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_hsFilter.fcl:6]: In the OTS implementation they share the same parameter. Here in the Offline this is just a test stand while we are waiting for the real DB to be up and running. These tables are not touched frequently, so I think that we can wait for the DB implementation and avoid complicating the system with another prolog file.

File: Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_tcFilter.fcl (hunk 2 above)
Comment (gianipez) [reply Trigger/data/cprDeM_highP_stopTarg/main_cprDeM_highP_stopTarg_tcFilter.fcl:2]: same comment as above

File: Trigger/data/tprDeP_highP_stopTarg/main_tprDeP_highP_stopTarg_tcFilter.fcl (hunk 3 above)
Comment (gianipez) [reply Trigger/data/tprDeP_highP_stopTarg/main_tprDeP_highP_stopTarg_tcFilter.fcl:2]: done

**Review by brownd1978 — APPROVED** [review]

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**